easybuild/easyconfigs/g/GPAW/GPAW-1.2.0-customize.patch

```diff
@@ -0,0 +1,28 @@
+--- customize.py	2017-02-24 15:10:13.635016617 +0100
++++ /home/camp/mikst/Desktop/customize.py	2017-02-24 15:19:02.267012659 +0100
+@@ -32,15 +32,21 @@
+ # mpicompiler = 'mpicc'  # use None if you don't want to build a gpaw-python
+ # mpilinker = 'mpicc'
+ # platform_id = ''
+-# scalapack = False
++scalapack = True
++libraries = ['readline',
++             'gfortran',
++             'scalapack',
++             'openblas',
```

Comment:
really new to easybuild, but is this the recommended way to go? (At least, I was not able to find another .eb file which use os.getenv('SCALAPACK_STATIC_LIBS'))